src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -353,6 +353,81 @@ To reiterate how the script system loads script files
 
 ## Docker Images
 
+The NUbots codebase uses Docker images to containerise the build process.
+Two images are created, one intended for running code on your local machine named `generic`, and another internded for running code on the robots named `nuc7i7bnh`.
```

Comment:
s/internded/intended
You really need to invest in a spell checker for your IDE :P